src/utils.jl

```diff
@@ -62,3 +75,73 @@ Base.getindex(::IdentityMap, index) = index
 function sparse_array_representation(func::MOI.AbstractFunction, num_variables)
     return sparse_array_representation(func, num_variables, IdentityMap())
 end
+
+"""
+    struct SparseVectorAffineFunction{T} <: MOI.AbstractVectorFunction
+        terms::SparseMatrixCSC{T,Int64}
+        constants::Vector{T}
+    end
+
+The vector-valued affine function ``A x + b``, where:
+
+* ``A`` is the sparse matrix given by `terms`
+* ``b`` is the vector `constants`
+"""
+struct SparseVectorAffineFunction{T} <: MOI.AbstractVectorFunction
+    terms::SparseMatrixCSC{T,Int64}
```

Comment:
Int should be sufficient everywhere no, since these are indices

```suggestion
        terms::SparseMatrixCSC{T,Int}
        constants::Vector{T}
    end

The vector-valued affine function ``A x + b``, where:

* ``A`` is the sparse matrix given by `terms`
* ``b`` is the vector `constants`
"""
struct SparseVectorAffineFunction{T} <: MOI.AbstractVectorFunction
    terms::SparseMatrixCSC{T,Int}
```